### PR TITLE
Quality of life changes

### PR DESCRIPTION
**Pull request by duke605**
Testing may need to be done. This was done in Notepad++. I'm more setup for discord.py

### CODE REVIEW

File: src/structures/GuildMember.js [hunk 1]
```diff
@@ -174,6 +174,15 @@ class GuildMember {
...
     return this.user.id;
   }
+  
+  /**
+  * The nickname of the member. Otherwise their username if nickname not set
+  * @type {string}
+  * @readonly
+  */
+  get displayName() {
+	return this.nickname || this.user.username
```
Comment (GusCaplan) [inline src/structures/GuildMember.js]: fix indentation

File: src/structures/User.js [hunk 2]
```diff
@@ -96,6 +96,26 @@ class User {
...
     return Constants.Endpoints.avatar(this.id, this.avatar);
   }
+  
+  /**
+  * A link to the user's default avatar
+   * @type {string}
+   * @readonly
+   */
+  get defaultAvatarURL() {
+	defaultAvatars = Object.values(Constants.DefaultAvatars)
```
Comment (GusCaplan) [inline src/structures/User.js]: fix indentation here

File: src/structures/User.js [hunk 3]
```diff
@@ -96,6 +96,26 @@ class User {
...
+	return Constants.Endpoints.assets(defaultAvatars[defaultAvatar] + '.png')
+  }
+  
+  /**
+  * A link to the user's avatar if they have one. Otherwise a link to their default avatar will be returned
+  * @type {string}
+  * @readonly
+  */
+  get displayAvatarURL() {
+	return this.avatarURL() || this.defaultAvatarURL()
```
Comment (GusCaplan) [inline src/structures/User.js]: indentation

File: src/util/Constants.js [hunk 4]
```diff
@@ -243,6 +245,14 @@ exports.MessageTypes = {
   6: 'PINS_ADD',
 };
 
+exports.DefaultAvatars = {
+	BLURPLE: '6debd47ed13483642cf09e832ed0bc1b',
```
Comment (GusCaplan) [inline src/util/Constants.js]: indentation

File: src/structures/GuildMember.js (hunk 1 above)
Comment (GusCaplan) [inline src/structures/GuildMember.js]: also commas

File: src/structures/User.js (hunk 2 above)
Comment (GusCaplan) [inline src/structures/User.js]: commas

File: src/structures/User.js (hunk 3 above)
Comment (GusCaplan) [inline src/structures/User.js]: COMMAS

File: src/util/Constants.js [hunk 5]
```diff
@@ -69,6 +69,7 @@ exports.Errors = {
 
 const PROTOCOL_VERSION = exports.PROTOCOL_VERSION = 6;
 const API = exports.API = `https://discordapp.com/api/v${PROTOCOL_VERSION}`;
+const HOST = exports.HOST = `https://discordapp.com`
```
Comment (GusCaplan) [inline src/util/Constants.js]: COMMA

**Comment by GusCaplan** [comment]
you need to do this on the indev branch

**Comment by Programmix** [comment]
Seems to be that `displayName` is completely useless. I see no reason to simplify such an already simple operation.

File: src/structures/User.js (hunk 3 above)
Comment (Programmix) [inline src/structures/User.js]: Also, you can't invoke getters.

**Pull request closed without merging**